.jenkins/library/vars/helpers.groovy

```diff
@@ -15,17 +18,18 @@ def WaitForAptLock() {
         i=0
         echo "Checking for locks..."
         # Check for locks
-        while fuser /var/lib/dpkg/lock > /dev/null 2>&1 ||
-              fuser /var/lib/dpkg/lock-frontend > /dev/null 2>&1 ||
-              fuser /var/lib/apt/lists/lock > /dev/null 2>&1; do
+        while sudo fuser /var/lib/dpkg/lock > /dev/null 2>&1 ||
+              sudo fuser /var/lib/dpkg/lock-frontend > /dev/null 2>&1 ||
+              sudo fuser /var/lib/apt/lists/lock > /dev/null 2>&1 ||
+              sudo ps aux | grep -E "[a]pt|[d]pkg"; do
             # Wait up to 600 seconds to lock to be released
             if (( i > 600 )); then
                 echo "Timeout waiting for lock."
                 exit 1
             fi
             echo "Waiting for apt/dpkg locks..."
-            i=\${i++}
-            sleep 1
+            i=\$((\$i+5))
```

Comment:
If it's not too much trouble, it might be a good idea to keep the sleep interval in a variable

Reply:
Good idea, will implement when I get back to this PR